reference/conanfile/methods.rst

```diff
@@ -128,3 +128,35 @@ direct
 
 If the dependency is a direct one, that is, it has explicitly been declared by the current
 recipe, or if it is a transitive one.
+
+
+validate_build()
```

Comment:
The reference of methods is a mess. But I prefered to copy that to not lose it when we improve it.